packages/website/package.json

```diff
@@ -55,7 +56,7 @@
     "typescript": "*"
   },
   "devDependencies": {
-    "@docusaurus/module-type-aliases": "^3.2.1",
+    "@docusaurus/module-type-aliases": "^3.7.0",
```

Comment:
I don't know why these are complaining either, but they shouldn't be part of this PR. #11262 is bumping them anyway.